lib/utils/utils_test.go

```diff
@@ -491,22 +491,25 @@ func (s *UtilsSuite) TestMarshalYAML(c *check.C) {
 }
 
 // TestReadToken tests reading token from file and as is
-func (s *UtilsSuite) TestReadToken(c *check.C) {
+func TestReadToken(t *testing.T) {
 	tok, err := ReadToken("token")
-	c.Assert(tok, check.Equals, "token")
-	c.Assert(err, check.IsNil)
+	require.Equal(t, tok, "token")
+	require.NoError(t, err)
 
 	_, err = ReadToken("/tmp/non-existent-token-for-teleport-tests-not-found")
-	fixtures.ExpectNotFound(c, err)
+	fixtures.AssertNotFound(t, err)
 
-	dir := c.MkDir()
+	_, err = ReadToken("./non-existent-token-for-teleport-tests-not-found")
+	fixtures.AssertNotFound(t, err)
+
+	dir := t.TempDir()
 	tokenPath := filepath.Join(dir, "token")
 	err = ioutil.WriteFile(tokenPath, []byte("shmoken"), 0644)
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	tok, err = ReadToken(tokenPath)
-	c.Assert(err, check.IsNil)
-	c.Assert(tok, check.Equals, "shmoken")
+	require.NoError(t, err)
+	require.Equal(t, tok, "shmoken")
 }
```

Comment:
Might be nice to add a few other test scenarios here:

1. `~./token`
2. `./token`
3. `../xyz/token`
4.  `./path/to/token`

Misc variations on those.